.github/workflows/Check_CSV.yml

```diff
@@ -0,0 +1,45 @@
+# Currently always using COMMA as csv delimiter
+# Could be extended for usage of other delimiters if required
+# (s. https://github.com/Clever/csvlint)
+name: Check if CSV files are valid
+
+on: workflow_call
+
+permissions:
+  contents: read
+
+jobs:
+  csvlint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+
+    - name: Install Ruby #(csvlint.io is a Ruby gem)
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.0'
+
+    - name: Install CSV Lint
+      run: gem install csvlint
+
+    - name: Lint all CSV files in the repository
+      shell: bash {0}
```

Comment:
Avoid passing the default "-e" argument to the bash (so that the action does not stop on the first invalid csv file, but checks all files)